mlflow/server/handlers.py

```diff
@@ -550,6 +550,18 @@ def _list_registered_models():
     return _wrap_response(response_message)
 
 
+@catch_mlflow_exception
+def _search_registered_models():
+    request_message = _get_request_message(SearchRegisteredModels())
+    registered_models = _get_model_registry_store().search_registered_models(
```

Comment:
This should return a `PagedList` and the handler should probably pass on the `next_page_token` here